firmware/bootloader/src/Makefile

```diff
@@ -116,12 +116,12 @@ endif
 
 # Define project name here
 PROJECT = bootloader
-PROJECT_DIR = ..
+PROJECT_DIR = ../..
 
 # Imported source files and paths
 CHIBIOS = $(PROJECT_DIR)/ChibiOS
 # todo: looks like 'CHIBIOS_CONTRIB' path is universal shall we defined it only once?
-CHIBIOS_CONTRIB = $(CHIBIOS)/../ChibiOS-Contrib
+CHIBIOS_CONTRIB = $(PROJECT_DIR)/ChibiOS-Contrib
```

Comment:
can it be $(CHIBIOS)/../ChibiOS-Contrib ?

Reply:
> can it be $(CHIBIOS)/../ChibiOS-Contrib ?

Yes. But Chibios and Chibios-Contrib are two independent units. Why should one be related to another?